setup.py

```diff
@@ -2,25 +2,32 @@
 # -*- coding: utf-8 -*-
 import os
 import sys
+import logging
 from typing import TYPE_CHECKING
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 if TYPE_CHECKING:
     from typing import Iterable, Set
 
 MAGPIE_ROOT = os.path.abspath(os.path.dirname(__file__))
-MAGPIE_MODULE_DIR = os.path.join(MAGPIE_ROOT, 'magpie')
+MAGPIE_MODULE_DIR = os.path.join(MAGPIE_ROOT, "magpie")
 sys.path.insert(0, MAGPIE_MODULE_DIR)
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
-from magpie import __meta__             # noqa: F401
+LOGGER = logging.getLogger("magpie.setup")
+if logging.StreamHandler not in LOGGER.handlers:
+    LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+LOGGER.setLevel(logging.INFO)
+LOGGER.info("starting setup")
 
-with open('README.rst') as readme_file:
+from magpie import __meta__  # noqa: F401
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Import "from magpie import __meta__" should be placed at the top of the module](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)